blockchain/types/src/avr_history.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Types for reading and writing historical Attestation Verification
+//! Reports from MobileCoin consensus enclaves
+
+use alloc::{string::String, vec::Vec};
+use hex::ToHex;
+use mc_attest_core::VerificationReport;
+use mc_crypto_keys::Ed25519Public;
+use mc_common::HashMap;
+use toml::{map::Map, value::Value};
+use url::Url;
+
+/// Object representing AVR history for all consensus nodes
+pub struct AVRHistoricalRecords {
```

Comment:
How about `AvrRecords`?

```suggestion
pub struct AvrRecords {
```

Reply:
Sure